src/components/nav/Nav.js

```diff
@@ -1,7 +1,84 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
+import './Nav.scss';
 
 const Nav = () => {
-  return <div>dsadasd</div>;
+  const inners = [
+    {
+      id: '1',
+      url: '/',
+      value: '인기상품',
+    },
+    {
+      id: '2',
+      url: '/',
+      value: '신상품',
+    },
+    {
+      id: '3',
+      url: '/',
+      value: '온라인 한정사이즈',
+    },
+    {
+      id: '4',
+      url: '/',
+      value: '가격 인하',
+    },
+    {
+      id: '5',
+      url: '/',
+      value: '기간 한정',
+    },
+    {
+      id: '6',
+      url: '/',
+      value: '의복잡화 아울렛',
+    },
+    {
+      id: '7',
+      url: '/',
+      value: '생활잡화 아울렛',
+    },
+  ];
+
+  const innerList = inners.map(inner => {
+    return (
+      <li key={inner.id}>
+        <Link to={inner.url} title={inner.value}>
+          {inner.value}
+        </Link>
+      </li>
+    );
+  });
```

Comment:
컴포넌트화가 아니고 같은 컴포넌트안에서 변수로 선언하신 이유가 있을까요?
nav컴포넌트 return부분에서는 깔끔해 보여도 연관성을 생각했을 때는 컴포넌트화를 시키거나 return 안에서 map을 돌려주는게 더 연관성 있고 유지보수하기도 좋을 것 같아요!
그리고, 로직이 아닌 단순 리턴이라면 {} 와 return은 생략가능합니다!
```suggestion
  const innerList = inners.map(inner => (
      <li key={inner.id}>
        <Link to={inner.url} title={inner.value}>
          {inner.value}
        </Link>
      </li>
    ));
```